src/types.jl

```diff
@@ -116,13 +116,44 @@ struct Ellipsoid
     e′²::Float64      # Second eccentricity squared
 end
 
-#auxiliary function
+"""
+    ellipsoid(a::BigFloat, b::BigFloat)
+
+Compute `a` and `b` to create an ellipsoid with a mathematical formula.
+
+# Examples
+```julia-repl
+julia> OpenStreetMapX.ellipsoid(BigFloat(a), BigFloat(b))
+1
+```
+
+**Arguments**
+* `a::BigFloat` : vertical axis of ellipsoid;
+* `b::BigFloat` : horizontal axis of ellipsoid;
+
+"""
 function ellipsoid(a::BigFloat, b::BigFloat)
     e² = (a^2 - b^2) / a^2
     e′² = (a^2 - b^2) / b^2
     OpenStreetMapX.Ellipsoid(a, b, e², e′²)
 end
-#constructor
+
+"""
+    Ellipsoid(; a::Float64 = NaN, b::Float64= NaN, f_inv::Float64= NaN)
+
+Constructor to create an ellipsoid high `a` and wide `b` using an inverse function `f_inv`.
+
+# Examples
+```julia-repl
+julia> OpenStreetMapX.Ellipsoid(BigFloat(a), BigFloat(b), BigFloat(b))
```

Comment:
This example is not useful and the code does not work.
BigFloat is not Float64. The cosntructor takes 4 argument (3 are accepted if these are named arguments)
Note that the apostrophe `;` works slightly differently in Python and Julia